### PR TITLE
feat: remove auth section

### PR DESCRIPTION
**Pull request by anastasiia-developer**
## What/Why/How?
feat: remove auth section 
## Screenshots (optional)

Screenshot 2022-05-27 at 18 43 36
Screenshot 2022-05-27 at 18 44 49

## Security unexpanded 

Screenshot 2022-05-27 at 18 40 10

## Security expanded

Screenshot 2022-05-27 at 18 40 20

## See more
Screenshot 2022-05-27 at 18 42 26

## See less

Screenshot 2022-05-27 at 18 42 38

## Check yourself

- [ ] Code is linted
- [ ] Tested
- [ ] All new/updated code is covered with tests

### CODE REVIEW

**Review by Oprysk — APPROVED** [review]

File: src/components/__tests__/SecurityRequirement.test.tsx [hunk 1]
```diff
@@ -1,28 +1,156 @@
...
-const options = new RedocNormalizedOptions({});
-describe('Components', () => {
-  describe('SecurityRequirement', () => {
-    describe('SecurityRequirement', () => {
-      it("should render 'None' when empty object in security open api", () => {
-        const parser = new OpenAPIParser(
-          { openapi: '3.0', info: { title: 'test', version: '0' }, paths: {} },
-          undefined,
-          options,
+const spec: OpenAPISpec = {
```
Comment (AlexVarchuk) [inline src/components/__tests__/SecurityRequirement.test.tsx:17]: Maybe better move spec to `fixtures`?

Comment (AlexVarchuk) [reply src/components/__tests__/SecurityRequirement.test.tsx:17]: Does it cover all security types, which we support?
``` 
  oauth2: 'OAuth2',
  apiKey: 'API Key',
  http: 'HTTP',
  openIdConnect: 'OpenID Connect',
```

**Review by AlexVarchuk — CHANGES_REQUESTED** [review]
Should we deprecate `noAutoAuth`? What do you think? @anastasiia-developer  @RomanHotsiy @Oprysk

File: src/components/__tests__/SecurityRequirement.test.tsx (hunk 1 above)
Comment (anastasiia-developer) [reply src/components/__tests__/SecurityRequirement.test.tsx]: > Maybe better move spec to fixtures?
Done 😄

File: src/services/OpenAPIParser.ts [hunk 2]
```diff
@@ -53,7 +47,7 @@ export class OpenAPIParser {
     private options: RedocNormalizedOptions = new RedocNormalizedOptions({}),
   ) {
     this.validate(spec);
-    this.preprocess(spec);
+    // this.preprocess(spec);
```
Comment (anastasiia-developer) [inline src/services/OpenAPIParser.ts:50]:
```suggestion
```

**Comment by zalesky** [comment]
Are these screenshots updated? I see too much vertical gap between `petstore_auth` `api_key` `Gitlub_Oauth2AuthorizationCodeWithPKCE` (Screenshot1)

File: src/components/SecurityRequirement/OAuthFlow.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+import { SeeMore } from '../SeeMore/SeeMore';
+import { Markdown } from '../Markdown/Markdown';
+
+export interface OAuthFlowProps {
+  type: string;
+  flow: OpenAPISecurityScheme['flows'][keyof OpenAPISecurityScheme['flows']];
+  RequiredScopes?: JSX.Element;
+}
+
+export class OAuthFlow extends React.PureComponent<OAuthFlowProps> {
```
Comment (zalesky) [inline src/components/SecurityRequirement/OAuthFlow.tsx:13]: It would be great to use functional component here

Comment (anastasiia-developer) [reply src/components/SecurityRequirement/OAuthFlow.tsx]: done

Comment (Oprysk) [reply src/components/SecurityRequirement/OAuthFlow.tsx]: @anastasiia-developer  Previously it was a `PureComponent` after refactoring it to a function component we need to keep the performance in the rendering perspective. So we might use React `memo`  HOC here, what do you think?

Comment (anastasiia-developer) [reply src/components/SecurityRequirement/OAuthFlow.tsx]: yes, we might
I'm going to add that

Comment (anastasiia-developer) [reply src/components/SecurityRequirement/OAuthFlow.tsx]: Done

**Review by zalesky — APPROVED** [review]

**Pull request merged**